docs/source/deployment/knative.rst

```diff
@@ -167,7 +185,8 @@ Validate prediction server with sample data
 ===========================================
 
 
-For this guide, our kubernetes cluster run on minikube, we will get the appropriate ip from minikube and the port from istio
+For this guide, the kubernetes cluster run on minikube, Get the appropriate ip from
```

Comment:
~For this guide~, ~the appropriate ip~

Mention `minikube` setup at the `Prerequisites` section

change this to:

Find the IP address and exposed port of the deployed Knative service: